### PR TITLE
Fix ennemy update static misstake

### PR DESCRIPTION
**Pull request by guillaumeAbel**

### CODE REVIEW

File: assets/data/player/stats.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "max_life": 100.0,
     "level max": 10.0,
-    "max_energy": 1000.0,
-    "xp max": 100.0,
+    "max_energy": 1100.0,
+    "xp max": 120.0,
     "actual xp": 0.0,
-    "actual lvl": 0.0
+    "actual lvl": 1.0
```
Comment (romainpanno) [inline assets/data/player/stats.json:7]: C 0

File: assets/data/player/stats.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "max_life": 100.0,
     "level max": 10.0,
-    "max_energy": 1000.0,
-    "xp max": 100.0,
+    "max_energy": 1100.0,
```
Comment (romainpanno) [inline assets/data/player/stats.json:4]: C 1000

File: assets/data/player/stats.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+    "xp max": 120.0,
```
Comment (romainpanno) [inline assets/data/player/stats.json:5]: C 100

**Review by romainpanno — CHANGES_REQUESTED** [review]

File: assets/data/player/stats.json (hunk 1 above)
Comment (guillaumeAbel) [reply assets/data/player/stats.json:7]: lessgo

File: assets/data/player/stats.json (hunk 2 above)
Comment (guillaumeAbel) [reply assets/data/player/stats.json:4]: lesssgo

**Review by romainpanno — APPROVED** [review]

**Pull request merged**